### PR TITLE
MUMPFL-81 Cleanup phone number fields

### PR DESCRIPTION
**Pull request by timlevett**
+ Changes all non-zero phone numbers to optional
+ Adds trash button to non-zero phone numbers
http://goo.gl/S2URML
+ Removes "add another" after 3 are on the screen
http://goo.gl/RT5U3O
+ Allows you to submit an empty phone number, as long as zero is filled out
http://goo.gl/m1BIPH
+ Doesn't save said zero phone number
+ Doesn't show an empty phone number in the view mode
http://goo.gl/5SO10N

### CODE REVIEW

File: my-profile-webapp/src/main/webapp/my-app/lec/partials/emergency.html [hunk 1]
```diff
@@ -21,27 +24,31 @@
...
-                <select id="phoneType{{$index}}" name="phoneType{{$index}}" class="form-control" required ng-model="contact.phoneNumbers[$index].type" ng-class="{ 'has-error' : emergencyForm.phoneType0.$invalid && emergencyForm.phoneType0.$touched }">
+                <select id="phoneType{{$index}}" name="phoneType{{$index}}" class="form-control" ng-required='$index === 0' ng-model="contact.phoneNumbers[$index].type" ng-class="{ 'has-error' : emergencyForm.phoneType0.$invalid && emergencyForm.phoneType0.$touched }">
                     <option value="mobile" selected="selected">Mobile</option>
                     <option value="home">Home</option>
                     <option value="work">Work</option>
                     <option value="other">Other</option>
                 </select>
+                
+            </div>
+            <div class='col-xs-2 no-padding'><button aria-label="Delete Phone Number" class='btn btn-default' style='top : 25px; margin-left: 10px;' ng-click='contact.phoneNumbers.splice($index, 1)' ng-if="$index !== 0"><i class='fa fa-trash'></i></a></div>
```
Comment (apetro) [inline my-profile-webapp/src/main/webapp/my-app/lec/partials/emergency.html]: Nice `aria-label`.

**Comment by apetro** [comment]
:+1:

**Comment by vertein** [comment]
:+1:

**Comment by jhanstra** [comment]
:+1:

**Pull request merged**